src/Features/ExternalAccess/HotReload/Microsoft.CodeAnalysis.ExternalAccess.HotReload.csproj

```diff
@@ -0,0 +1,42 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <RootNamespace>Microsoft.CodeAnalysis.ExternalAccess.HotReload</RootNamespace>
+    <TargetFramework>$(NetRoslyn)</TargetFramework>
+
+    <!-- NuGet -->
+    <IsPackable>true</IsPackable>
+    <PackageId>Microsoft.CodeAnalysis.ExternalAccess.HotReload</PackageId>
```

Comment:
Should this package be added to PublishData.json?

Reply:
Yes, thanks for catching it